### ISSUE TITLE
`ns dev` hangs at package.load with embedded Postgres resource

### ISSUE THREAD

**Issue by szferi** [namespacelabs/foundation#996]
[ns-test2.tar.gz](https://github.com/namespacelabs/foundation/files/10196021/ns-test2.tar.gz)
The attached workspace file helps to reproduce the bug. 
After `ns prepare local` and `ns dev .` the package.load hangs on `namespacelabs.dev/foundation/library/oss/postgres`

On the same setup, the `multitier/03-withresources/` example works as intended.

Logs: 
```
Dec  9 15:29:11.752 UTC workspace.load-workspace dir="/Users/szferi/Projects/ns/ns-test2" waited=19us took=457us
Dec  9 15:29:11.752 UTC workspace.load-workspace dir="/Users/szferi/Projects/ns/ns-test2" waited=2us took=120us
Dec  9 15:29:11.753 UTC module.download name="namespacelabs.dev/foundation" version="420eb5dbff4de60ec2eeaeb9bc931ae5aec6d824" waited=2us took=20us
Dec  9 15:29:11.753 UTC workspace.load-workspace dir="/Users/szferi/Library/Caches/ns/module/namespacelabs.dev/foundation/420eb5dbff4de60ec2eeaeb9bc931ae5aec6d824" waited=8us took=498us
Dec  9 15:29:11.757 UTC package.load namespacelabs.dev/foundation/library/runtime waited=2us took=2ms
Dec  9 15:29:11.759 UTC package.load namespacelabs.dev/foundation/library/database/postgres waited=2us took=2ms
Dec  9 15:29:11.760 UTC package.load namespacelabs.dev/foundation/library/oss/postgres/prepare/database waited=10us took=224us
Dec  9 15:29:11.760 UTC package.load namespacelabs.dev/foundation/library/oss/postgres/prepare/cluster waited=11us took=404us
Dec  9 15:29:11.762 UTC package.load namespacelabs.dev/foundation/library/oss/postgres/server waited=2us took=1ms
  debug version check: current v0.0.108, build time seconds:1670461177, min API 0
  debug location parsing: . -> moduleName: "ns-test2" rel: "." expando: false isRef: false
Dec  9 15:29:12.281 UTC ns.check-updated waited=6us took=529ms
  debug version check: got v0.0.108, build time: 2022-12-08 01:01:22 +0000 UTC, new: false
 => package.load-server ns-test2 (8.5s) 
 ==> package.load ns-test2 (8.5s) 
 ===> package.load namespacelabs.dev/foundation/library/oss/postgres (8.5s) 
[+] 8.5s 3 actions running.
```

OS: Mac OS Venture 13.0.1
```
> ns version
version v0.0.108 (commit f8a72958db5d217d4591d94ec88a26ec52d8668b)
  commit date 2022-12-08T00:59:37Z
  architecture darwin/arm64
  internal api 46 (cache=1 tools=4)
 ````

**Comment by hugosantos** [comment]
I've confirmed this is an issue in 0.0.108, but not at head. Releasing 109.

**Comment by szferi** [comment]
I confirm that the issue has been resolved with v0.0.109.
